react/what-is-props/what-is-props.md

```diff
@@ -0,0 +1,131 @@
+# What is props?
+
+---
+
+If you are beginner in react you must have heard about `props` a lot, so let's understand, how is it so important to understand.
```

Comment:
instead of `how is it so important to understand`, we can say `let's understand why it's important`.

Try to say things in less words with more meaning, also the word `understand` came twice.